resolvers/functions/creatorCheck.js

```diff
@@ -1,5 +1,5 @@
 const creatorCheck = (context, org) => {
-  const isCreator = org.creator === context.userId;
+  const isCreator = org.creator == context.userId;
   if (!isCreator) {
```

Comment:
== is loosely check. which is a bad practice
use === ,  match types of the variables